vagrant-provision.sh

```diff
@@ -0,0 +1,14 @@
+:
+
+sudo apt-get update && sudo rm -rf /var/lib/apt/lists/*
```

Comment:
Welcome to LEAN @rterbush ! Awesome to see the first work on a Vagrant file. If possible could you bring this setup file inline with the Docker setup? As is - it'll just compile LEAN and miss the other installations necessary for it to fully work (minimally python, libraries, java and the IB versions we host in the CDN).
 https://github.com/QuantConnect/Lean/blob/master/DockerfileLeanFoundation

Reply:
I will have a look. Currently working through getting Lean working in my environment as time permits, so I will add to this as I make progress.